src/main/java/de/bas/deploymentmanager/logic/domain/stage/control/StageServiceImpl.java

```diff
@@ -19,15 +22,19 @@
 public class StageServiceImpl implements StageService {
     private final Logger log = LoggerFactory.getLogger(this.getClass());
 
+    private final ProjectRepository projectRepository;
```

Comment:
Inject von Projectservice ist nicht erlaubt an dieser Stelle. 
Die beiden befinden sich in Unterschiedlichen Domains 

Funktionalität sollte im LoadStageFlow gebündelt werden

Reply:
Verstehe ich nicht ganz, denn da sind ja schon andere Repos drin...